### DISCUSSION TITLE
Need help with DKIM and SPF for e-mail through my domain

### DISCUSSION THREAD

**Discussion by SoroushJSP** [vercel/community#2897] [Help]
### Summary

I used webhostingcanada to acquire my domain name, but vercel to deploy it. I changed the nameservers which allowed the site to actually load on the domain perfectly, but i have an form that can't send me e-mails. I reached out to WHC support and they said I needed to add some DNS settings, DKIM and SPF, so I added those to the DNS sub-domain page on vercel. As far as I can tell, on my WHC panel they're now 'valid', but I still can't receive e-mails. 

Is there anything more than adding those two TXT records to my vercel that I need to do?

### Steps to Reproduce

Vercel DNS settings:

![image](https://github.com/vercel/community/assets/133778194/dac931e0-6af3-4c18-a5b4-500bee43b89f)

WHC Panel:
![image](https://github.com/vercel/community/assets/133778194/9c2cf0a8-d95c-40db-9bf2-c4bb284dbbf0)

**Comment by smaeda-ks** [comment]
@SoroushJSP are you still having an issue? Or, did it resolve after DNS propagation?

**Reply by SoroushJSP** [reply]
Yes I am, DNS propagation did not resolve
